waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -596,7 +596,8 @@
                   result (http-utils/http-request
                            http-client
                            log-bucket-url
-                           :query-string query-string)
+                           :query-string query-string
+                           :spnego-auth true)
```

Comment:
Can we default this to false and configure it to be true?

Reply:
We discussed this offline, and for now I've added a comment both here and on the curl command in the waiter-init script. Ideally we'd make this configurable, but for now we'll settle for the comment since the current code works in both kerberized and non-kerberized environments.

Reply:
Diff: https://github.com/twosigma/waiter/compare/134f7c59a432bb516f1643cc934290ecf32c0da4..76966f795fc3626bb2744ed86bd193c2a6ec229e